test/e2e/main_test.go

```diff
@@ -96,9 +96,9 @@ func TestMain(m *testing.M) {
 	}
 
 	// The DEPLOY_KBS is exported then provisioner will install kbs before installing CAA
-	shouldDeployKbs := true
-	if os.Getenv("DEPLOY_KBS") != "yes" {
-		shouldDeployKbs = false
+	shouldDeployKbs := false
+	if os.Getenv("DEPLOY_KBS") == "true" {
+		shouldDeployKbs = true
```

Comment:
Also can we shift L121-L124 under this condition `if shouldDeployKbs {` on L138 ? It was shifted as part of review but i think we should move this condition down.